public/js/tas.js

```diff
@@ -1,10 +1,14 @@
 $(function(){
+  // listens for screen size inserts an <hr> tag
+  if ($(window).width() > 600) {
+    $('#placeholder').html('<hr>');
+  }
   // read and dynamically insert total signatures from Firebase, currently reading total
   //   number of backers
   var backerCount = new Firebase('https://standwithduwamish.firebaseio.com/backerCount');
   backerCount.on("value", function(snapshot){
     var signatureCount = snapshot.val();
-    $('#signatures').html(signatureCount); // set initial value of users/signatures
+    $('#sigs-count').html(signatureCount); // set initial value of users/signatures
```

Comment:
This bit is more safely done with .text() instead of .html(). snapshot.val() isn't user-generated content so it's not vital here, but still a best practice.